po/fr.po

```diff
@@ -259,21 +258,21 @@ msgstr "Vous devez vous connecter pour suivre quelqu’un"
 msgid "You need to be logged in order to edit your profile"
 msgstr "Vous devez vous connecter pour modifier votre profil"
 
-msgid "By {{ link_1 }}{{ link_2 }}{{ link_3 }}{{ name | escape }}{{ link_4 }}"
-msgstr ""
-"Par {{ link_1 }}{{ link_2 }}{{ link_3 }}{{ name | escape }}{{ link_4 }}"
+#, fuzzy
+msgid "By {0}"
```

Comment:
is it normal for `{0}` to be translated by `{0}{1}{2}`?

Reply:
No, it is a translation I forgot to update I think.